content/en/_index.md

```diff
@@ -18,8 +18,8 @@ It provides high performance compression and encoding schemes to handle complex
 
 
 {{< blocks/section color="white" type="row">}}
-{{% blocks/feature icon="fab fa-jira" title="Parquet Specification" url="https://github.com/apache/parquet-format" %}}
-Browse the Parquet Specification
+{{% blocks/feature icon="fab fa-jira" title="Documentation" url="docs" %}}
+Browse project documentation including the format specification.
```

Comment:
sending people immediately to the spec from the homepage was a bit jarring -- I suggest linking them to the intro page on the docs (which now also has a link to the actual spec, along with a clearer description)